src/BudgetManager/BudgetManager.CQRS/Handlers/UserHandlers/GetUserPayersHandler.cs

```diff
@@ -0,0 +1,24 @@
+﻿using BudgetManager.CQRS.Queries.UserQueries;
+using BudgetManager.Model;
+using BudgetManager.Shared.DataAccess.MongoDB.BaseImplementation;
+using MediatR;
+
+namespace BudgetManager.CQRS.Handlers.UserHandlers
+{
+    public class GetUserPayersHandler : IRequestHandler<GetUserPayersQuery, IEnumerable<string>?>
+    {
+        private readonly IBaseRepository<User> _dataAccess;
+
+        public GetUserPayersHandler(IBaseRepository<User> dataAccess)
+        {
+            _dataAccess = dataAccess;
+        }
+
+        public async Task<IEnumerable<string>?> Handle(GetUserPayersQuery request, CancellationToken cancellationToken)
+        {
+            var user = await _dataAccess.FindByIdAsync(request.userId, cancellationToken);
+
+            return user is not null ? user.Payers : null;
```

Comment:
You can use a null conditional operator here to shorten the code length if you like:

`return user?.Payers`

if the user is null, it immediately returns null without throwing a NullReferenceException, otherwise, it accesses the Payers property and returns it.